tests/docx/body_xml_tests.py

```diff
@@ -1162,6 +1169,8 @@ def _linked_blip(relationship_id):
 def _blip(attributes):
     return xml_element("a:blip", attributes)
 
+def _ext(width='0', height='0'):
```

Comment:
The general style of the codebase is to default arguments to `None`. Although this in case, I'd suggest that the defaults can be removed altogether, and the `ext` element be left out when not explicitly set.